src/providers/authentication/SIOPv2Provider.ts

```diff
@@ -1,8 +1,8 @@
 import {CheckLinkedDomain, VerifiedAuthorizationRequest} from '@sphereon/did-auth-siop';
 import {ConnectionTypeEnum, IDidAuthConfig} from '@sphereon/ssi-sdk-data-store';
-import {OpSession, VerifiablePresentationWithDefinition} from '@sphereon/ssi-sdk-did-auth-siop-authenticator';
-import {getKey} from '@sphereon/ssi-sdk-did-auth-siop-authenticator/dist/session/functions';
-import {VerifiableCredentialsWithDefinition} from '@sphereon/ssi-sdk-did-auth-siop-authenticator/src/types/IDidAuthSiopOpAuthenticator';
+import {OpSession, VerifiableCredentialsWithDefinition, VerifiablePresentationWithDefinition} from '@sphereon/ssi-sdk-did-auth-siop-authenticator';
+import {OID4VP} from '@sphereon/ssi-sdk-did-auth-siop-authenticator/dist/session/OID4VP'; // FIXME we should fix the export of these objects
```

Comment:
Yeah should be fixed in latest feature branch which has the package name changed as well. That SDK will be released at roughly the same time as the wallet, but it will not be in the first version of the wallet itself